hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/tasks/TestReconTaskControllerImpl.java

```diff
@@ -88,17 +88,29 @@ public void testConsumeOMEvents() throws Exception {
     when(omUpdateEventBatchMock.isEmpty()).thenReturn(false);
     when(omUpdateEventBatchMock.filter(Collections.singleton("MockTable")))
         .thenReturn(omUpdateEventBatchMock);
+
+    long startTime = System.currentTimeMillis();
     reconTaskController.consumeOMEvents(
         omUpdateEventBatchMock,
         mock(OMMetadataManager.class));
 
     verify(reconDBUpdateTaskMock, times(1))
         .process(any());
+    long endTime = System.currentTimeMillis();
+
+    ReconTaskStatusDao dao = new ReconTaskStatusDao(sqlConfiguration);
+    ReconTaskStatus reconTaskStatus = dao.findById("MockTask");
+    long taskTimeStamp = reconTaskStatus.getLastUpdatedTimestamp();
+
+    Assert.assertTrue(startTime <= taskTimeStamp
```

Comment:
Can we add sequence number test here just like you have added in testReInitializeTasks()?